template.distillery/demo_calendar.eliom

```diff
@@ -21,6 +21,18 @@ let%server service =
 let%client service = ~%service
 
 (* A reactive value containing the currently selected date *)
+(* NOTE: in this example, we define a shared signal on the server side. It can
+   be accessed when the server generates the first page (so, declaring it
```

Comment:
Not updatable because the implementation does not use a server-side reactive signal at all (but only the fixed value of the signal). If curious, see [`Eliom_shared`](https://github.com/ocsigen/eliom/blob/master/src/lib/eliom_shared.eliom#L272).

For whatever is missing from [the documentation page](http://ocsigen.org/eliom/6.2/manual/clientserver-react), I will be happy to review PRs :).